collections/appusers.go

```diff
@@ -50,8 +53,16 @@ func (u *AppUsers) GetOrCreate(ctx context.Context, projectID, deviceID string)
 	user = &AppUser{
 		ID:        deviceID,
 		ProjectID: projectID,
+		Created:   time.Now().Unix(),
 	}
-	if err := u.threads.ModelCreate(ctx, u.storeID.String(), u.GetName(), user); err != nil {
+	user.StoreID, err = u.threads.NewStore(ctx)
```

Comment:
This associates a dedicated store with the new user. Again, this will likely change, but not sure how yet.